staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -173,17 +253,8 @@ func GRPCStreamInterceptor(interceptor grpc.StreamServerInterceptor) Option {
 	}
 }
 
-// NodeV1alpha4 explicitly chooses whether the DRA gRPC API v1alpha4
-// gets enabled.
-func NodeV1alpha4(enabled bool) Option {
-	return func(o *options) error {
-		o.nodeV1alpha4 = enabled
-		return nil
-	}
-}
-
 // NodeV1beta1 explicitly chooses whether the DRA gRPC API v1beta1
-// gets enabled.
+// gets enabled. True by default.
```

Comment:
Is a boolean flexible enough for this? How does this change as we add new API versions? What happens when this is set to `false`?

Reply:
When explicitly set to false, the server cannot start at the moment because there is no other API version left.

I could have removed it, but then I would have to bring it back if we ever add another alternative.

Reply:
I added the same comment as for the listener options:
> This is used in Kubernetes for end-to-end testing. The default should
> be fine for DRA drivers.